src/pages/publication/PublicProfile.tsx

```diff
@@ -1,5 +1,97 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { addNotification } from '../../redux/actions/notificationActions';
+import { PublishedPublicationPreview } from '../../types/publication.types';
+import { CircularProgress } from '@material-ui/core';
+import styled from 'styled-components';
+import { getMyPublications } from '../../api/publicationApi';
+import PublishedPublicationList from './PublishedPublicationList';
+import i18n from '../../translations/i18n';
+import Card from '../../components/Card';
+import { useTranslation } from 'react-i18next';
+import { RootStore } from '../../redux/reducers/rootReducer';
+import NormalText from '../../components/NormalText';
+import { ORCID_BASE_URL } from '../../utils/constants';
+import LabelTextLine from './../../components/LabelTextLine';
+import Heading from '../../components/Heading';
 
-const PublicProfile: FC = () => <div>public profile...</div>;
+const StyledWrapper = styled.div`
+  text-align: center;
+  padding: 0.5rem;
+`;
+
+const StyledUserInfo = styled.div`
+  display: flex;
+  background-color: ${props => props.theme.palette.background.default};
+  align-content: flex-start;
+  width: 100%;
+  padding: 0.5rem;
+`;
+
+const StyledPicture = styled(Card)`
+  width: 10rem;
+`;
+
+const StyledUser = styled(Card)`
+  flex-grow: 4;
+`;
+
+const PublicProfile: FC = () => {
+  const dispatch = useDispatch();
+  const [isLoading, setIsLoading] = useState(true);
+  const [publications, setPublications] = useState<PublishedPublicationPreview[]>([]);
+  const { t } = useTranslation();
+  const user = useSelector((store: RootStore) => store.user);
```

Comment:
hm... toucher borti forrige kommentar her. Må sikkert gå til backend for å hente all data om en bruker gitt ID her siden man skal kunne se andres profil også?
I så fall kan vi godt ta det som en egen sak, så blir dette en placeholder for nå

Reply:
ikke helt public nei :-)  Foreslår å lage en egen task med api-oppslag (mock) på vilkårlig user

Reply:
siden er vel heller ikke synlig om man ikke er logget inn ?

Reply:
Ikke slik featuren er skrevet